README.md

```diff
@@ -1,3 +1,39 @@
-# Device Controller
+# Yurt-device-controller
 
-This repository contains the device-controller that leverages EdgeX Foundry to manage edge devices.
+This repository contains three CRD/controllers, Device, DeviceService and DeviceProfile:
+
+- The `DeviceProfile` defines a type of devices using same kind of protocol, which includes some generic information like the manufacturer's name, the device description, and the device model. DeviceProfile also defines what kind of resources (e.g., temperature, humidity) this type of device provided and how to read/write these resources.
+
+- The `DeviceService` defines the way of how to connect a device to the OpenYurt, like the URL of the device. The `DeviceService` can not exist alone. Every `DeviceService` must associate with a `DeviceProfile`.
+
+- The `Device` is used to refer to a sensor, actuator, or IoT "thing", it gives the detailed definition of a specific device, like which `DeviceProfile` it belongs to and which `DeviceService` it used to connect to the system.
+
+For details of the design, please see the [document](https://github.com/openyurtio/openyurt/blob/master/docs/proposals/20210310-edge-device-management.md).
+
+## Getting Start
+
+To use the yurt-device-controller, you need to deploy the OpenYurt cluster in advance and meet the following two conditions:
+
+- Deploy the yurt-app-manager；
+- Deploy an Edgex Foundry instance in a NodePool by using yurt-edgex-manager；
```

Comment:
please add access link for yurt-edgex-manager

Reply:
the access link is added, please review